docs/staging-production-deployment.md

```diff
@@ -63,6 +63,13 @@ Copy `env.example` in the `autodeployment` directory to `.env` and add the appro
   - deployment type (`staging` or `production`)
   - unsplash client id
 
+#### 3.1- Autodeployment server - Module: Slack Integration
+
+- env variables
+  - slack webhook url (#telescope's url Slack Incoming Webhooks)
+  - interval one (120000)
+  - interval two (10000)
```

Comment:
```suggestion
# Explanation here about what these intervals are for
  - interval one (120 * 1000)
  - interval two (10 * 1000)
```
Splitting the numbers like this^ helps you understand what they mean:

Reply:
Sure.